### PR TITLE
Replace legacy vue components in Item Card [SCI-10177]

### PR DESCRIPTION
**Pull request by G-Chubinidze**
Jira ticket: [SCI-10177](https://scinote.atlassian.net/browse/SCI-10177)

### What was done
_Replaced legacy vue components_

[SCI-10177]: https://scinote.atlassian.net/browse/SCI-10177?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by aignatov-bio — CHANGES_REQUESTED** [review]
Did you test? 
In new one we don't have - `change` action
And we don't have params - withEditCursor, withClearButton, isLoading, searchPlachrolder, customClass, optionsClassName

**Comment by G-Chubinidze** [comment]
> Did you test? In new one we don't have - `change` action And we don't have params - withEditCursor, withClearButton, isLoading, searchPlachrolder, customClass, optionsClassName

Yes, no errors though, will double check and remove unnecessary params

File: app/javascript/vue/repository_item_sidebar/repository_values/RepositoryListValue.vue [hunk 1]
```diff
@@ -4,22 +4,17 @@
       {{ colName }}
     </div>
     <div>
-      <select-search
+      <SelectDropdown
         v-if="permissions?.can_manage && !inArchivedRepositoryRow"
         ref="DropdownSelector"
         @change="changeSelected"
         @update="update"
```
Comment (aignatov-bio) [inline app/javascript/vue/repository_item_sidebar/repository_values/RepositoryListValue.vue:11]: And how update will work?

**Review by aignatov-bio — CHANGES_REQUESTED** [review]

File: app/javascript/vue/repository_item_sidebar/repository_values/RepositoryListValue.vue (hunk 1 above)
Comment (G-Chubinidze) [reply app/javascript/vue/repository_item_sidebar/repository_values/RepositoryListValue.vue:11]: What do you mean?

**Review by artoscinote — APPROVED** [review]

**Pull request merged**